src/main/java/appeng/util/InventoryAdaptor.java

```diff
@@ -67,10 +69,26 @@ public static InventoryAdaptor getAdaptor(final EntityPlayer te) {
         return null;
     }
 
-    // return what was extracted.
-    public abstract ItemStack removeItems(int amount, ItemStack filter, IInventoryDestination destination);
```

Comment:
This changes the signature of a method widely used in addons, which seems to be out of scope of what the PR is intended to do.